### ISSUE TITLE
Request: add few more tags

### ISSUE THREAD

**Issue by Tecktron** [tiangolo/uwsgi-nginx-docker#56]
Would love to see some more generic tags.

My thoughts are the addition of:
`stable` - the most stable version (so latest may not be stable, IE for bleeding edge versions of OS, python, uwsgi, etc.)
`alpine-stable` - the stable version of alpine
`alpine-latest` - the latest alpine version (which may also not be stable)

Thanks for the consideration and the images.

**Comment by tiangolo** [comment]
Thanks for the comment. The problem is that I'm not the authority to decide which version is stable or not. It would depend on each application.

For example, Python 3.7 is considered "stable". But there are still some popular packages that are not fully compatible with it.

I prefer not to add those additional tags as they might end up being misleading for developers.

**Comment by Tecktron** [comment]
Right so to me, a stable docker image would mean the stable versions that play nicely together, making the image as a whole a stable image. I find confusion to be a relative term based on what you know, like I found it confusing at first when I pulled the alpine 3.7 image and it didn’t really work. Then I realized that 3.7 wasn’t stable with uwsgi and so I had to pull the 3.6 version. 

Ultimately I was just hoping for an easier auto-upgrade path than having to watch the repo then update all my Dockerfiles when the alpine image with uwsgi and 3.x became stable. 

All that said, your repo, your rules, and I respect that. I’m not going to argue beyond my given use case, which I leave for your further consideration. I am, however, going to close this since it seems we have a difference of opinion in what stable means with regards to docker images.

Thank you for this repo.
